### PR TITLE
Removes the security bombsuit / EOD closet

### LINKED ISSUE #28971: A bunch of suit sprites are broken
so far I found that the paper mache wizard robes are broken, so are a few of the lesser used chameleon gear armor items.

### PR DESCRIPTION
**Pull request by ShizCalev**
This removes an unused security variant of the bombsuit.

The suit's sprites were deleted by accident by commit 37068f64dbc3c443393df45df145ddb02d2203fa

That was 4 and a half years ago.

Literally nobody has used it in four and a half years to notice the suit's sprite didn't exist.

Closes #28971 (Was the only one I found out of the 100~ variants that I tried from the chameleon suit that was broken. Someone else fixed the paper wizard robes a while back too from the looks of it)

### CODE REVIEW

**Comment by Bawhoppen** [comment]
I remember seeing it like last month there's no way

**Comment by ShizCalev** [comment]
These were it's sprites

image

**Comment by ShizCalev** [comment]
Here's the suit on the ground.
image

**Comment by Bawhoppen** [comment]
Oh I was thinking of the biosuit.
Yea this can go

File: code/game/objects/structures/crates_lockers/closets/utility_closets.dm [hunk 1]
```diff
@@ -156,18 +156,6 @@
 	new /obj/item/clothing/shoes/sneakers/black( src )
 	new /obj/item/clothing/head/bomb_hood( src )
 
-
-/obj/structure/closet/bombclosetsecurity
```
Comment (Mark9013100) [inline code/game/objects/structures/crates_lockers/closets/utility_closets.dm]: IT WASN'T EVEN A SUBTYPE HNNNNNNNNNNNNGGGGG

**Comment by praisenarsie** [comment]
@shizcalev change it to fixes #number instead of closes so it actually closes issue

**Comment by Jordie0608** [comment]
`Closes` also works
https://help.github.com/articles/closing-issues-using-keywords/

**Comment by jughu** [comment]
IM USING THIS

**Review by Cyberboss — APPROVED** [review]

**Comment by FantasticFwoosh** [comment]
"Improve don't remove" only works when it doesn't suit you eh? 😆 

Im in favour of just giving it a touch up, re-implementing the sprites pre-change or supplying new ones, its a preparation thing in the case of a syndicate bio-attack or if a criminal is spreading a disease, if people would remember to use it.

But then again infection code being so poorly maintained means that bio suits are only partially helpful versus blood transmitted diseases when you're harmbatoning someone to death.

**Comment by ShizCalev** [comment]
@FantasticFwoosh you do realize this was a _bombsuit_ and not a _biosuit_, right?

**Comment by Bawhoppen** [comment]
Replying to Fwoosh just encourages him

**Pull request closed without merging**